go/vt/wrangler/testlib/backup_test.go

```diff
@@ -866,6 +866,87 @@ func TestDisableActiveReparents(t *testing.T) {
 	assert.True(t, destTablet.FakeMysqlDaemon.Running)
 }
 
+func TestDoNotDemotePrimaryAfterXtrabackup(t *testing.T) {
+	// TODO: need to figure out how to set backupEngineImplementation = "xtrabackup"
```

Comment:
I've been poking around and I'm not sure how to do this. There's another test suite in vtctlbackup package that I see uses backup_utils to force it to use xtrabackup instead of buildinbackup